react/features/base/react/components/web/NavigateSectionListItem.js

```diff
@@ -73,19 +73,21 @@ export default class NavigateSectionListItem<P: Props>
             <Container
                 className = 'navigate-section-list-tile'
                 onClick = { onPress }>
-                <Text
-                    className = 'navigate-section-tile-title'>
-                    { title }
-                </Text>
-                <Text
-                    className = 'navigate-section-tile-body'>
-                    { date }
-                </Text>
-                <Text
-                    className = 'navigate-section-tile-body'>
-                    { duration }
-                </Text>
-
+                <Container className = 'navigate-section-list-tile-info'>
+                    <Text
+                        className = 'navigate-section-tile-title'>
+                        { title }
+                    </Text>
+                    <Text
+                        className = 'navigate-section-tile-body'>
+                        { date }
+                    </Text>
+                    <Text
+                        className = 'navigate-section-tile-body'>
+                        { duration }
+                    </Text>
+                </Container>
+                { elementAfter || null }
```

Comment:
If we add the elementAfter, as a completely constructed element, then we don't need the action props I've added previously. I'm going to remove them.